employees/models.py

```diff
@@ -15,8 +15,17 @@
 from users.models import CustomUser
 
 
+class TaskActivityTypeQuerySet(models.QuerySet):
+    def filter_default_task_activities(self) -> QuerySet:
+        return self.filter(is_default=True)
+
+
 class TaskActivityType(models.Model):
     name = models.CharField(max_length=TaskActivityTypeConstans.TASK_ACTIVITIES_MAX_LENGTH.value)
+    is_default = models.BooleanField(default=False)
+    projects = models.ManyToManyField(Project, related_name="project_activities")
```

Comment:
So how it is going to work now? One task activity type can be assigned to multiple projects? What if in one project manager will remove or rename it?

Reply:
@rwrzesien Yes, as you said one task activity could be assigned to multiple projects, what is more, in my case I thought to give permission to change name of task activity only for admin. If PM needs to create some special task He could to this in manage view and it will be automaticly added to project. @kbeker Maybe in this situations we could change it to `other`?

Reply:
@Szymiks OK makes sense to limit access only for admin, but what if there will be the same type used in two projects, and in one manager would like to rename it? Even if he create a new one, old reports will still be using old one.

Reply:
@rwrzesien I understand, but how often can such a case happen? If there is a problem with the name for everyone, all you need to do is change the name of activity  by admin, but if only one manager need to change this he could create new one and use it. It seems to me that this problem will only occur after the other person's feedback, so will old reports with old activity be did it matter? However this is only my small opinion, I think that someone decisional should give his point of view.

Reply:
I agree that it not for us to decide on requirements, I just wanted to point out the problem that could be caused with current implementation. But if it is agreed that it will be resolved in the way which you have described, it is ok for me.